docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/google-tag-manager-custom-template/v3-tags/tag-template-guide/index.md

```diff
@@ -168,6 +168,82 @@ Using the **Context Entities** table allows you to attach [custom context entiti
 
 You can also choose to [set the True Timestamp](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/index.md#setting-the-true-timestamp) with this field. The format must be UNIX time in milliseconds.
 
+### Using Plugins
```

Comment:
Nit: maybe this would be better as a separate subpage? Just to avoid the page getting too large.

Reply:
Good call, it is a rather sudden wall of text on this page!